components/DetailsPageHighlights/detailspagehighlights.scss

```diff
@@ -0,0 +1,183 @@
+.bx--artifact-details-highlight-container {
+
+  & .bx--createArtifactFlexGroupColumn {
+    margin-bottom: rem(35px);
+  }
+
+  & .bx--createArtifactFlexGroupColumn {
+    display: flex;
+    flex-direction: column;
+
+    &.bx--hide {
+      display: none;
+    }
+  }
+
+  & .bx--title-label {
+    font-size: 1.55rem;
```

Comment:
Elsewhere we use `rem(pixelvalue)` for font-size. Would be good to standardize around that